skore/tests/unit/sklearn/plot/precision_recall_curve/test_comparison_estimator.py

```diff
@@ -187,6 +187,89 @@ def test_wrong_kwargs(pyplot, fixture_name, request, pr_curve_kwargs):
         display.plot(pr_curve_kwargs=pr_curve_kwargs)
 
 
+@pytest.mark.parametrize("with_average_precision", [False, True])
+def test_frame_binary_classification(
+    binary_classification_data, with_average_precision
+):
+    """Test the frame method with binary classification comparison data."""
+    estimator, X_train, X_test, y_train, y_test = binary_classification_data
+    estimator_2 = clone(estimator).set_params(C=10).fit(X_train, y_train)
+    report = ComparisonReport(
+        reports={
+            "estimator_1": EstimatorReport(
+                estimator,
+                X_train=X_train,
+                y_train=y_train,
+                X_test=X_test,
+                y_test=y_test,
+            ),
+            "estimator_2": EstimatorReport(
+                estimator_2,
+                X_train=X_train,
+                y_train=y_train,
+                X_test=X_test,
+                y_test=y_test,
+            ),
+        }
+    )
+    display = report.metrics.precision_recall()
+    df = display.frame(with_average_precision=with_average_precision)
+
+    expected_index = ["estimator_name"]
+    expected_columns = ["threshold", "precision", "recall"]
+    if with_average_precision:
+        expected_columns.append("average_precision")
+
+    check_frame_structure(df, expected_index, expected_columns)
+    assert df["estimator_name"].nunique() == 2
+
+    if with_average_precision:
+        for (_), group in df.groupby(["estimator_name"]):
+            assert group["average_precision"].nunique() == 1
+
+
+@pytest.mark.parametrize("with_average_precision", [False, True])
+def test_frame_multiclass_classification(
+    multiclass_classification_data, with_average_precision
+):
+    """Test the frame method with multiclass classification comparison data."""
+    estimator, X_train, X_test, y_train, y_test = multiclass_classification_data
+    estimator_2 = clone(estimator).set_params(C=10).fit(X_train, y_train)
+    report = ComparisonReport(
+        reports={
+            "estimator_1": EstimatorReport(
+                estimator,
+                X_train=X_train,
+                y_train=y_train,
+                X_test=X_test,
+                y_test=y_test,
+            ),
+            "estimator_2": EstimatorReport(
+                estimator_2,
+                X_train=X_train,
+                y_train=y_train,
+                X_test=X_test,
+                y_test=y_test,
+            ),
+        }
+    )
+    display = report.metrics.precision_recall()
+    df = display.frame(with_average_precision=with_average_precision)
+
+    expected_index = ["estimator_name", "label"]
+    expected_columns = ["threshold", "precision", "recall"]
+    if with_average_precision:
+        expected_columns.append("average_precision")
+
+    check_frame_structure(df, expected_index, expected_columns)
+    assert df["estimator_name"].nunique() == 2
+    assert df["label"].nunique() == len(report.reports_[0].estimator_.classes_)
```

Comment:
Like for the other test.

```suggestion

    check_frame_structure(df, expected_index, expected_columns)
    assert df["estimator_name"].nunique() == 2
```